spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  recipe = FactoryBot.create(:recipe)
```

Comment:
I’d prefer if you use the ‘let’ notation

Reply:
Used let instead